igvm/vm.py

```diff
@@ -652,6 +660,33 @@ def rename(self, new_hostname):
 
             self.start(transaction=transaction)
 
+    def aws_rename(self, new_hostname) -> None:
```

Comment:
Why no type hint for self and new_hostname ?

Reply:
added